common/src/main/java/com/wynntils/features/user/overlays/ShamanTotemTrackingFeature.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Model;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.TextRenderSetting;
+import com.wynntils.gui.render.TextRenderTask;
+import com.wynntils.gui.render.VerticalAlignment;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wynn.event.TotemRemovedEvent;
+import com.wynntils.wynn.event.TotemSummonedEvent;
+import com.wynntils.wynn.objects.ShamanTotem;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.world.entity.decoration.ArmorStand;
+import net.minecraft.world.scores.PlayerTeam;
+import net.minecraft.world.scores.Scoreboard;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ShamanTotemTrackingFeature extends UserFeature {
```

Comment:
Add `@FeatureInfo(category = FeatureCategories.OVERLAYS)`

Reply:
Actually, don't you think this categorization breaks down a bit? Maybe you're only interested in using this for the totem highlighting, not the overlay timers?

Just a random thought...